bioimage_manifest.yaml

```diff
@@ -0,0 +1,8 @@
+name: Example Models
+description: A repository for example models
+
+models:
+  - UNet2d.model.yaml
+
+transforms:
```

Comment:
we agreed on the use of `transformation` over `transform`, therefore I'd suggest to use `transformations` here.